doc/tutorial.rst

```diff
@@ -273,7 +273,8 @@ example but go ahead and `read up`_ on them if you want.
  It would really be a pain to specify that regex on the command line all the time, particularly if we're using many other options.
  That's what a configuration file is for. We can configure our Pylint to
  store our options for us so we don't have to declare them on the command line.  Using a configuration file is a nice way of formalizing your rules and
- quickly sharing them with others. Invoking ``pylint --generate-rcfile`` will create a sample rcfile with all the options set and explained in comments.
+ quickly sharing them with others. Invoking ``pylint --generate-toml-config`` will create a sample ``.toml`` section with all the options set and explained in comments.
+ This can then be added to your ``pyproject.toml`` file.
```

Comment:
(or any toml configuration file that you're linking with the ``--rc-file`` option)

Reply:
Want me to add that? I think following the (slow) trend towards unification in configuration files is good and a contribution we can make as project to the general python community.

Reply:
I think it's better to use a toml file that can be copy paster in pyproject.toml in the future than to stick to an ini pylintrc so if someone is recluctant to use pyproject.toml right now he or she can still use a toml with another name and we won't have to maintain the tool to generate ini file.

Reply:
Added a new line!